package.json

```diff
@@ -186,7 +186,6 @@
     "@visx/react-spring": "^2.12.2",
     "@visx/responsive": "^2.10.0",
     "@visx/shape": "^2.11.1",
-    "@walletconnect/ethereum-provider": "^2.7.8",
```

Comment:
this can be removed now that @web3-react/walletconnect-v2 has been updated to include a newer version